packages/actor-query-operation-path-seq/lib/ActorQueryOperationPathSeq.ts

```diff
@@ -0,0 +1,54 @@
+import {ActorAbstractPath} from "@comunica/actor-abstract-path";
+import {
+  ActorQueryOperation,
+  Bindings, IActorQueryOperationOutput, IActorQueryOperationOutputBindings,
+  IActorQueryOperationTypedMediatedArgs,
+} from "@comunica/bus-query-operation";
+import {ActorRdfJoin, IActionRdfJoin} from "@comunica/bus-rdf-join";
+import {ActionContext, Mediator} from "@comunica/core";
+import {IMediatorTypeIterations} from "@comunica/mediatortype-iterations";
+import {termToString} from "rdf-string";
+import {Algebra} from "sparqlalgebrajs";
+
+/**
+ * A comunica Path Seq Query Operation Actor.
+ */
+export class ActorQueryOperationPathSeq extends ActorAbstractPath {
+
+  public readonly mediatorJoin: Mediator<ActorRdfJoin,
+    IActionRdfJoin, IMediatorTypeIterations, IActorQueryOperationOutput>;
+
+  constructor(args: IActorQueryOperationPathSeq) {
+    super(args, Algebra.types.SEQ);
+  }
+
+  public async runOperation(path: Algebra.Path, context: ActionContext)
+    : Promise<IActorQueryOperationOutputBindings> {
+    const predicate = <Algebra.Seq> path.predicate;
+    const blank = this.generateBlankNode(path);
+    const blankName = termToString(blank);
+
+    const left = ActorQueryOperation.getSafeBindings(await this.mediatorQueryOperation.mediate({
```

Comment:
We can optimize this by moving the await's to a higher level, so that left and right can run in parallel.